src/frontend/helpers/getPlaystreakArgsFromQuestData.ts

```diff
@@ -0,0 +1,33 @@
+import { PlayStreakEligibility } from '@hyperplay/ui'
+import { Quest, UserPlayStreak } from 'common/types'
+
+// this is initialized when the overlay is started for this game
+const dateTimeCurrentSessionStartedInMsSinceEpoch = Date.now()
+
+export function getPlaystreakArgsFromQuestData(
+  questMeta: Quest,
+  questPlayStreakData: UserPlayStreak | undefined,
+  userIsLoggedIn?: boolean
+): PlayStreakEligibility {
+  console.log(
+    'in getPlaystreakArgsFromQuestData quest meta ',
+    questMeta,
+    questPlayStreakData,
+    userIsLoggedIn
+  )
```

Comment:
do we want to add this to the logger instead?

Reply:
I should actually remove it. good catch